### ISSUE TITLE
The version number for the vic-machine-server included in the 1.3.0 RC2 OVA is not as expected

### ISSUE THREAD

**Issue by zjs** [vmware/vic-product#1170]
**VIC Product version:**

`vic-v1.3.0-rc2-2628-9e2a95b6.ova`

**Deployment details:**

Environment where the OVA is deployed

**Steps to reproduce:**

Deploy the OVA, use the UI to create a VCH.

**Actual behavior:**

The first line of the log is:
```
Nov 29 2017 00:16:04.133Z INFO Starting API-based VCH Creation. Version: "v1.2.0-rc1 git:18841ed-dirty build:2017/11/23@16:22:42 id:15115 runtime:go1.8.3"
```

**Expected behavior:**

The version number matches the version number for the `vic-machine-*` binaries included in the VIC engine bundle.

**Comment by zjs** [comment]
If this is strictly a build-time issue, this must be fixed for 1.3.0. If it is a runtime issue, a description of the root cause should be shared and the issue should be re-triaged.

**Comment by andrewtchin** [comment]
After the recent tags the version looks more correct `Nov 29 2017 19:44:03.562Z INFO  Starting Service. Version: "v1.3.0-rc1 git:be27555-dirty build:2017/11/27@21:00:32 id:15140 runtime:go1.8.3"`
I will investigate why the build is showing dirty in CI
